questions/includes/openSUSE/coding/ruby.yml

```diff
@@ -3,3 +3,19 @@ tree:
     - title: software.opensuse.org
       subtitle: a way to discover and install package on openSUSE distributions
       link: https://github.com/openSUSE/software-o-o/blob/master/CONTRIBUTING.md
+
+    - title: Travel Support Program
+      subtitle: Help people in the project attend events by improving tsp.opensuse.org 
+      link: https://github.com/openSUSE/travel-support-program
+
+    - title: Open Source Event Manager
+      subtitle: Use your Ruby skills to make our event management tool for FOSS conferences even better
+      link: https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md
+
+    - title: Open Build Service Frontend
+      subtitle: Contribute to the frontend of the Open Build Service; a generic system to build and distribute packages from source 
+      link: https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md
+      
+    - title: software.opensuse.org
```

Comment:
This is a duplicate, also it seems like the site doesn't support dots in the titles

Reply:
excellent. Happy we have something deployed. I deleted the duplicate and dots.